### PR TITLE
[C-1596] Fix android mobile-upload bugs

### PR DESCRIPTION
**Pull request by dylanjeffers**
### Description

Fixes various android-specific mobile-upload bugs
- Improves Switch sizing
- Fixes release date double render + colors (that's what all the .xml style/color changes are)
- Fixes demiBold font margins for android
- Fixes navigation object for newly moved fullscreenModals
- Fixes TouchableOpacity double click (seems like bug with react-native-gesture-handler)
- Hides the bottom-section of upload screen when keyboard present (since keyboard doesn't slide over for some android devices)
- Fixes Remix Pill text spacing

### CODE REVIEW

File: packages/mobile/src/screens/edit-track-screen/components/FormScreen.tsx [hunk 1]
```diff
@@ -32,6 +33,26 @@ export const FormScreen = (props: FormScreenProps) => {
   const { children, bottomSection, style: styleProp, ...other } = props
   const styles = useStyles()
   const navigation = useNavigation()
+  const [hideBottomSection, setHideBottomSection] = useState(false)
+
+  // // On android, explicitly hide bottomSection when keydoard shown.
```
Comment (sliptype) [inline packages/mobile/src/screens/edit-track-screen/components/FormScreen.tsx:38]: Double comment

File: packages/mobile/src/screens/edit-track-screen/components/FormScreen.tsx [hunk 2]
```diff
@@ -32,6 +33,26 @@ export const FormScreen = (props: FormScreenProps) => {
...
+  // // This will prevent users from accidently pressing "continue".
+  useLayoutEffect(() => {
+    if (Platform.OS === 'android') {
+      const showSubscription = Keyboard.addListener('keyboardDidShow', () => {
```
Comment (sliptype) [inline packages/mobile/src/screens/edit-track-screen/components/FormScreen.tsx:42]: Probably not useful in this case but we do have the keyboard state in the store

**Review by sliptype — APPROVED** [review]
Good stuff!

File: packages/mobile/src/screens/edit-track-screen/components/FormScreen.tsx (hunk 2 above)
Comment (dylanjeffers) [reply packages/mobile/src/screens/edit-track-screen/components/FormScreen.tsx:42]: oh yeah good call, let me check

Comment (dylanjeffers) [reply packages/mobile/src/screens/edit-track-screen/components/FormScreen.tsx]: its much cleaner now, great call

**Pull request merged**